week-5/Homework/mandatory/3-project/js/main.js

```diff
@@ -0,0 +1,65 @@
+let blueChanges = document.querySelector("#blueBtn");
+blueChanges.addEventListener("click", function () {
+  document.querySelector(".jumbotron").style.backgroundColor = "#588fbd";
+  document.querySelector(".jumbotron .btn").style.backgroundColor = "#ffa500";
+  document.querySelector(".jumbotron .btn-secondary").style.backgroundColor =
+    "black";
+  document.querySelector(".jumbotron .btn-secondary").style.color = "white";
+});
+
+let orangeChanges = document.querySelector("#orangeBtn");
+orangeChanges.addEventListener("click", function () {
+  document.querySelector(".jumbotron").style.backgroundColor = "#f0ad4e";
+  document.querySelector(".jumbotron .btn").style.backgroundColor = "#5751fd";
+  document.querySelector(".jumbotron .btn-secondary").style.backgroundColor =
+    "#31b0d5";
+  document.querySelector(".jumbotron .btn-secondary").style.color = "white";
+});
+
+let greenChanges = document.querySelector("#greenBtn");
+greenChanges.addEventListener("click", function () {
+  document.querySelector(".jumbotron").style.backgroundColor = "#87ca8a";
+  document.querySelector(".jumbotron .btn").style.backgroundColor = "black";
+  document.querySelector(".jumbotron .btn-secondary").style.backgroundColor =
+    "#8c9c08";
+});
+
+//Part 2
+
+let findButtom = document.getElementsByTagName("button")[4];
+console.log(findButtom);
+
+let email = document.getElementById("exampleInputEmail1");
+let name = document.getElementById("example-text-input");
+let text = document.getElementById("exampleTextarea");
+
+findButtom.addEventListener("click", function (e) {
+  e.preventDefault();
+  if (email.value.length <= 0 || !email.value.includes("@")) {
+    email.style.backgroundColor = "red";
+  } else {
+    email.style.backgroundColor = "white";
+  }
+  if (name.value.length <= 0) {
+    name.style.backgroundColor = "red";
+  } else {
+    name.style.backgroundColor = "white";
+  }
+  if (text.value.length <= 0) {
+    text.style.backgroundColor = "red";
+  } else {
+    text.style.backgroundColor = "white";
+  }
+
+  if (
+    email.value.length > 0 &&
+    email.value.includes("@") &&
+    name.value.length > 0 &&
+    text.value.length > 0
+  ) {
+    alert("Thank you for filling out this form");
+    email.value = "";
+    name.value = "";
+    text.value = "";
+  }
```

Comment:
Awesome work! The behaviour of the form is correct but there is always room for improvement🙂 

There is an easier way to select the button we need here. You can find a button with a type of submit like this:
```let submitButton = document.querySelector("button[type='submit']");```

I can also see several if else statements checking the same condition, and then you check them again from line 54. It could be extracted to a separate function responsible for the validation. Remember, keep your code [DRY (Don't Repeat Yourself)](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) - you will hear it a lot😀